package.json

```diff
@@ -20,11 +20,13 @@
   },
   "dependencies": {
     "@breejs/later": "^4.1.0",
+    "@types/ws": "8.5.4",
     "@valtech-sd/rules-js": "^1.0.0",
     "amqp-cacoon": "^3.2.0",
     "coronado-bridge": "^2.2.0",
     "lodash": "^4.17.21",
     "log4js": "^6.3.0",
+    "mqtt": "^4.3.8",
```

Comment:
I ran into some not so fun typescript version issues getting MQTT installed. 

I had to upgrade node/typescript slightly, they now align with the version our services use. 

I also had to downgrade the @types/node version on some of our services. We had v17 types while using node v16, mqtt / ws did not like this at all!

 Also had to pin @types/ws to 8.5.4. 8.5.11 requires typescript 4.8.